go/libraries/doltcore/remotestorage/chunk_store.go

```diff
@@ -42,6 +42,20 @@ import (
 	"github.com/dolthub/dolt/go/store/types"
 )
 
+var DownloadHedger *Hedger
+
+func init() {
+	// TODO: This does not necessarily respond well to changes in network
+	// conditions during the program's runtime.
+	DownloadHedger = NewHedger(
+		8,
+		NewMinStrategy(
+			1*time.Second,
+			NewPercentileStrategy(0, 60*60*1000, 4, 95.0),
```

Comment:
Duration is in nanoseconds right?  So this is 60*60 microseconds or 3.6 milliseconds?  A named constant using time.Duration constants such as time.Microsecond, or time.Millisecond would be clearer.  As it is I'm not sure if this is the intended value or not.